internal/service/federatedsettingsorgconfig/data_source_federated_settings_connected_orgs.go

```diff
@@ -130,20 +131,21 @@ func PluralDataSource() *schema.Resource {
 }
 func dataSourceMongoDBAtlasFederatedSettingsOrganizationConfigsRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
 	// Get client connection.
```

Comment:
knit: comment can be removed

Reply:
it's a knit, i try to remove as i go over it (boy scout) but ok also to do it in bulk in a different PR